plugins/thunderbolt/fu-thunderbolt-device.c

```diff
@@ -580,6 +580,8 @@ fu_thunderbolt_device_setup_retimer(FuDevice *device, GError **error)
 	fu_device_add_flag(device, FWUPD_DEVICE_FLAG_DUAL_IMAGE);
 	fu_device_add_flag(device, FWUPD_DEVICE_FLAG_INTERNAL);
 	fu_device_add_internal_flag(device, FU_DEVICE_INTERNAL_FLAG_NO_AUTO_REMOVE);
+	fu_device_add_flag(device, FWUPD_DEVICE_FLAG_SKIPS_RESTART);
+	fu_device_remove_flag(device, FWUPD_DEVICE_FLAG_REQUIRE_AC);
```

Comment:
what added it in the first place? I'd rather not add it in the first place rather than add it and take it away again.

Reply:
@hughsie this flags are inherited for all TBT devices , I adjusted this flags in fu_thunderbolt_device_setup_retimer 
added FWUPD_DEVICE_FLAG_SKIPS_RESTART as the restart is causing the retimers to de-enumerate from sysfs after nvm update { this leaves ports offline for ever} and wont properly exit.

I have removed FWUPD_DEVICE_FLAG_REQUIRE_AC since when ports go offline the PD gets suspended hence AC wont be available.